### PR TITLE
Adding jsonNode to format logs in json string

### PR DESCRIPTION
**Pull request by dgurjar-ch**
Adding jsonNode to format logs in json string

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: src/main/java/uk/gov/companieshouse/document/generator/consumer/document/service/impl/MessageServiceImpl.java [hunk 1]
```diff
@@ -123,24 +126,34 @@ private Map<String, Object> setCompletedDebugMap(DocumentGenerationCompleted com
...
-
+    private Map<String, Object> setStartedDebugMap(DocumentGenerationStarted started)  {
         Map<String, Object> startedParams = new HashMap<>();
-        startedParams.put(STARTED_DOCUMENT, started);
+        startedParams.put(STARTED_DOCUMENT, getObjectMapper(String.valueOf(started)));
 
         return startedParams;
     }
+
+    private JsonNode getObjectMapper(String logStatus){
```
Comment (dhawes-ch) [inline src/main/java/uk/gov/companieshouse/document/generator/consumer/document/service/impl/MessageServiceImpl.java:149]: This method should be named something more appropriate like getLogStatusTree
The catch block shouldn't be empty, do something with the exception or let it crash
And setting node = null is rough, what happens when the catch block let's the null fall through? I'd assume a vague hard to debug NPE, at the very least add a null check

Comment (dgurjar-ch) [reply src/main/java/uk/gov/companieshouse/document/generator/consumer/document/service/impl/MessageServiceImpl.java]: made changes

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Review by aduffy-ch — APPROVED** [review]

**Pull request merged**